rxnorm-integration/src/test/java/dev/ikm/tinkar/rxnorm/integration/RxnormDescriptionSemanticIT.java

```diff
@@ -0,0 +1,93 @@
+     */
+    @Test
+    public void testRxnormDescriptionSemantics() throws IOException {
+        String sourceFilePath = "../rxnorm-origin/";
+        String errorFile = "target/failsafe-reports/Rxnorm_Descriptions_not_found.txt";
+        String absolutePath = findFilePath(sourceFilePath, rxnormOwlFileName);
+        int notFound = processOwlFile(absolutePath, errorFile);
+
+        assertEquals(0, notFound, "Unable to find " + notFound + " Rxnorm Description semantics. Details written to " + errorFile);
+    }
+
+    @Override
+    protected boolean assertOwlElement(RxnormData rxnormData) {
+        if (rxnormData.getId() != null) {
+            // Generate UUID based on RxNorm ID
+            UUID conceptUuid = uuid(rxnormData.getId());
+
+//            EntityProxy.Concept concept = EntityProxy.Concept.make(PublicIds.of(conceptUuid));
+            EntityProxy.Semantic descSemantic =
+                    EntityProxy.Semantic.make(PublicIds.of(UuidT5Generator.get(UUID.fromString(namespaceString),
+                    (EntityProxy.Concept.make(PublicIds.of(conceptUuid))).publicId().asUuidArray()[0] + rxnormData.getRxnormName() + "DESC")));
+//            (concept.publicId().asUuidArray()[0] + rxnormData.getRxnormName() + "DESC")
+
+            StateSet state = StateSet.ACTIVE;
+            StampPositionRecord stampPosition = StampPositionRecord.make(timeForStamp, TinkarTerm.DEVELOPMENT_PATH.nid());
+            StampCalculator stampCalc = StampCoordinateRecord.make(state, stampPosition).stampCalculator();
+//            ConceptRecord entity = EntityService.get().getEntityFast(conceptUuid);
+//            SemanticRecord entity = EntityService.get().getEntity();
+//            SemanticRecord entity = EntityService.get().getEntityFast(conceptUuid);
+            SemanticRecord entity = EntityService.get().getEntityFast(descSemantic.uuids());
+
+            PatternEntityVersion latestDescriptionPattern = (PatternEntityVersion) Calculators.Stamp.DevelopmentLatest().latest(TinkarTerm.DESCRIPTION_PATTERN).get();
+//            Latest<ConceptVersionRecord> latest = stampCalc.latest(entity);
+            Latest<SemanticVersionRecord> latest = stampCalc.latest(entity);
+
+            if (latest.isPresent()) {
+                System.out.println("latest.isPresent(): " + latest.get().toString());
+
+                String textForDesc = latestDescriptionPattern.getFieldWithMeaning(TinkarTerm.TEXT_FOR_DESCRIPTION, latest.get());
+                Component descCaseSignificance = latestDescriptionPattern.getFieldWithMeaning(TinkarTerm.DESCRIPTION_CASE_SIGNIFICANCE, latest.get());
+                Component descType = latestDescriptionPattern.getFieldWithMeaning(TinkarTerm.DESCRIPTION_TYPE, latest.get());
+
+                return ((textForDesc.equals(rxnormData.getRxnormName())
+                                ||  textForDesc.equals(rxnormData.getRxnormSynonym())
```

Comment:
the return tests should be based on what the tranformation process is adding to the semantic.